demos/common_demo_header.hpp

```diff
@@ -0,0 +1,20 @@
+/**
+ * @file common_demo_header.hpp
+ * @author Maximilien Naveau (maximilien.naveau@gmail.com)
+ * @brief Contains some default tools for creating demos
+ * @version 0.1
+ * @date 2019-11-07
+ * 
+ * @copyright Copyright (c) 2019
+ * 
+ */
+
+#include "blmc_robots/common_header.hpp"
+
+namespace blmc_robots
+{
+
+
+
+
+}
```

Comment:
for reviewing it would be really useful if you create meaningful commits, which completely implement one small feature. if half is in one and the other half in another commit, it makes reviewing much harder.